src/gui/resources/translations/de.ts

```diff
@@ -0,0 +1,2749 @@
+
+Vergewissern Sie sich, dass die Geräte angeschlossen sind, und klicken Sie auf OK.</translation>
+    </message>
+    <message>
+        <location filename="../../settingswidget.cpp" line="211"/>
+        <location filename="../../settingswidget.cpp" line="231"/>
+        <source>Error executing ckb-next-dev-detect</source>
+        <translation>Fehler beim Ausführen von ckb-next-dev-detect</translation>
+    </message>
+    <message>
+        <location filename="../../settingswidget.cpp" line="211"/>
+        <source>An error occurred while trying to execute ckb-next-dev-detect.
+File not found or not executable.</source>
+        <translation>Beim Versuch, ckb-next-dev-detect auszuführen, ist ein Fehler aufgetreten.
+Datei nicht gefunden oder nicht ausführbar.</translation>
+    </message>
+    <message>
+        <location filename="../../settingswidget.cpp" line="216"/>
+        <source>Generating Report</source>
+        <translation>Bericht wird erzeugt</translation>
+    </message>
+    <message>
+        <location filename="../../settingswidget.cpp" line="227"/>
+        <source>An error occurred while trying to execute ckb-next-dev-detect.
+
+</source>
+        <translation>Beim Versuch, ckb-next-dev-detect auszuführen, ist ein Fehler aufgetreten.
+
+</translation>
+    </message>
+    <message>
+        <location filename="../../settingswidget.cpp" line="230"/>
+        <source>Return code %1</source>
+        <translation>Rückgabecode %1</translation>
+    </message>
+    <message>
+        <location filename="../../settingswidget.cpp" line="236"/>
+        <source>Select output directory</source>
+        <translation>Ausgabe-Ordner wählen</translation>
+    </message>
+    <message>
+        <location filename="../../settingswidget.cpp" line="239"/>
+        <source>Report generated successfully</source>
+        <translation>Bericht erfolgreich erzeugt</translation>
+    </message>
+    <message>
+        <location filename="../../settingswidget.cpp" line="239"/>
+        <source>The report has been generated successfully.</source>
+        <translation>Der Bericht wurde erfolgreich erzeugt.</translation>
+    </message>
+    <message>
+        <location filename="../../settingswidget.cpp" line="249"/>
+        <source>Error writing report</source>
+        <translation>Fehler beim Speichern des Berichts</translation>
+    </message>
+    <message>
+        <location filename="../../settingswidget.cpp" line="249"/>
+        <source>Could not write report to the selected directory.
+Please pick a different one and try again.</source>
+        <translation>Der Bericht konnte nicht in das ausgewählte Verzeichnis gespeichert werden.
+Bitte wählen Sie ein anderes Verzeichnis und versuchen Sie es erneut.</translation>
+    </message>
+    <message>
+        <location filename="../../settingswidget.cpp" line="268"/>
+        <source>Checking...</source>
+        <translation>Überprüfen...</translation>
```

Comment:
I might have already mentioned it, but I guess the progressive form would be "Überprüfe..."